Database/schema.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE games(
```

Comment:
Please rename tables as per required schema, e.g. "game" instead of "games":
![image](https://user-images.githubusercontent.com/106581139/220289024-d4fa988a-a7fb-4c2d-bc1c-c05cf2b7bf51.png)